packages/vue/src/generators/application/files/common/src/vue-shims.d.ts.template

```diff
@@ -0,0 +1,5 @@
+declare module '*.vue' {
```

Comment:
This is needed because `typecheck` never worked without Volar in VSCode. The shims here at least makes it pass even though nothing meaningful is checked. We already do the same for Vue libs.